md_api/src/main/java/com/wipro/ats/bdre/md/api/HaltDeploy.java

```diff
@@ -43,6 +37,11 @@ public HaltDeploy() {
     @Autowired
     DeployDAO deployDAO;
 
+    public HaltDeploy() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
+
     public HaltDeployInfo execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)